server/app.js

```diff
@@ -1,11 +1,13 @@
 const express = require('express');
+const path = require('path');
 const db = require('./db/index.js');
 
 const app = express();
 
 app.set('port', 3004);
```

Comment:
I believe it's good practice to set the port number via an environment variable.

Reply:
Yes I need to set up file to import setup variables